### PR TITLE
feat: add default `Simple` tab for transactions

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

This PR adds a default `Simple` tab for transactions. This will be useful for sidechains, so that some information is still displayed about their transactions even if there isn't full support for the transaction.

### Context of Change

It's useful for sidechain development - you can view information about your transaction without needing to go to the `Raw` tab.

### Type of Change

- [x] New feature (non-breaking change which adds functionality)

### TypeScript/Hooks Update

The new pages already use TS/hooks

## Before / After

Before:
image

image

After:
image

image

image

## Test Plan

Added tests to make sure everything is displayed properly.

### CODE REVIEW

File: src/containers/shared/components/Transaction/DefaultSimple.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+              )
+            }
+            return getRow(index.toString(), childValue, depth, index.toString())
+          })}
+        </>
+      </SimpleGroup>
+    )
+  }
+
+  if (key === 'Amount') {
```
Comment (ckniffen) [inline src/containers/shared/components/Transaction/DefaultSimple.tsx:66]: Would this this fall under the case on line 87?

Comment (mvadari) [reply src/containers/shared/components/Transaction/DefaultSimple.tsx:66]: No - this catches the case where `Amount` represents XRP

File: src/containers/shared/components/Transaction/DefaultSimple.tsx [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+        Object.keys(childValue).length === 1
+      ) {
+        const childKey = Object.keys(childValue)[0]
+        return <div key={childValue}>{processValue(childValue[childKey])}</div>
+      }
+      return <div key={childValue}>{processValue(childValue)}</div>
+    })
+  }
+
+  if (typeof value === 'object') {
```
Comment (ckniffen) [inline src/containers/shared/components/Transaction/DefaultSimple.tsx:66]: There should be a test for array and object values.

**Comment by ckniffen** [comment]
The coverage report. 
```DefaultSimple.tsx   |   64.91 |    39.13 |   64.29 |   63.64 | 47-50,54-62,67-70,93,105,128-150,161-164 ```

**Review by ckniffen — APPROVED** [review]

**Review by jonathanlei — APPROVED** [review]

**Pull request merged**